user_docs/en/userGuide.t2t

```diff
@@ -65,46 +74,224 @@ This applies to both original and modified copies of this software, plus any der
 For further details, you can [view the full license. https://www.gnu.org/licenses/old-licenses/gpl-2.0.html]
 For details regarding exceptions, access the license document from the NVDA menu under the "help" section.
 
-+ System Requirements +[SystemRequirements]
-- Operating Systems: all 32-bit and 64-bit editions of Windows 7, Windows 8, Windows 8.1, Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2008 R2.
- - For Windows 7, NVDA requires Service Pack 1 or higher.
- - For Windows Server 2008 R2, NVDA requires Service Pack 1 or higher.
-- at least 150 MB of storage space.
--
 
-+ Getting and Setting Up NVDA +[GettingAndSettingUpNVDA]
-If you have not yet got a copy of NVDA, you can download it from the [NV Access website NVDA_URL].
++ NVDA Quick Start Guide +[NVDAQuickStartGuide]
+
+This quick start guide contains three main sections: downloading, initial setup, and running NVDA.
+These are followed by information on adjusting preferences, participating in the community and getting help.
+The information in this guide is condensed from other parts of the NVDA User Guide.
+Please refer to the full User Guide for more detailed information on each topic.
+
+++ Downloading NVDA +[GettingAndSettingUpNVDA]

```

Comment:
The number of `+`s must be the same on either side, this is causing several headings to not render properly.

```suggestion
++ Downloading NVDA ++[GettingAndSettingUpNVDA]
```